scripts/content.js

```diff
@@ -24,7 +24,7 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
     brands = resp;
```

Comment:
you don't need this line - on the line below just do `resp.result.map`

Reply:
I don't understand why its wrong?